DocumentDataAPI/DocumentDataAPITests/Data/Repositories/NpgDocumentRepositoryIntegrationTests.cs

```diff
@@ -19,7 +18,45 @@ public NpgDocumentRepositoryIntegrationTests()
         _logger = new Logger<NpgDocumentRepository>(new NullLoggerFactory());
         TestHelper.DeployDatabaseWithTestData();
     }
+    
+    [Fact]
+    public void Get_ReturnsCorrectRows()
+    {
+        // Arrange
+        NpgDocumentRepository repository = new(_connectionFactory, _logger);
+        DocumentModel model = new("Maja Lærke Maach",
+                                  DateTime.Parse("2022-10-07T13:40:00"),
+                                  1,
+                                  "https://www.dr.dk/nyheder/seneste/iran-haevder-mahsa-amini-doede-af-organsvigt",
+                                  1,
+                                  "",
+                                  "Iran hævder, at Mahsa Amini døde af organsvigt",
+                                  0);
```

Comment:
De her models kan jo laves som memberdata, så skal det ikke gentages så mange gange. Se tests for wordratio for inspiration